api_mongoDB.go

```diff
@@ -28,10 +28,9 @@ import (
 )
 
 var Client *mongo.Client = nil
-var dbName string
 var pools = map[string]map[string]int{}
 
-func SetMongoDB(setdbName string, url string) {
+func SetMongoDB(url string) {
```

Comment:
I would highly recommend changing the design of this library to something that will be easier to use for the long term.

I think the library should not keep its own client and always operate on it, because it still limits the library to a single Mongo server. 

A better approach would be to have a function that returns a MongoDBClient:

```go
type MongoDBClient struct {
    client *mongo.Client
}

func NewMongoDBClient(url, dbname string) (*MongoDBClient, error) {
    // Create the client
    ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
    client, err := mongo.Connect(ctx, options.Client().ApplyURI(url))
    defer cancel()
    if err != nil {
        return nil, err
    }
    
    return &MongoDBClient{client: client}, nil
}
```

And then, we could change the different functions in this file and make them methods on a `*MongoDBClient`:

```go
func (m *MongoDBClient) RestfulAPIGetOne(collname string, filter bson.M) map[string]interface{} {
    collection := m.client.Database(dbName).Collection(collName)
    // etc...
    return result
}
```

_None of the code in this comment is tested, just trying to show the idea_

Reply:
All the methods still require the database to be specified with each query. My suggestion was to have that in the returned client. However, I think we should follow @thakurajayL suggestion and use https://github.com/omec-project/util/tree/main/mongoapi

Reply:
PR is updated to define separate MongoDB clients for different databases. As written by @thakurajayL,  https://github.com/omec-project/util/tree/main/mongoapi will be used in webconsole and UDR projects.